sample/shared/package.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "shared",
+    "author": "Workleap",
+    "version": "0.0.0",
+    "description": "Shared package to showcase @squide.",
+    "license": "Apache-2.0",
+    "type": "module",
+    "main": "./dist/index.js",
+    "types": "./dist/index.d.ts",
+    "scripts": {
+        "dev": "tsup --config ./tsup.dev.ts"
+    },
+    "peerDependencies": {
+        "react": "*",
+        "react-dom": "*",
+        "react-router-dom": "*"
+    },
+    "devDependencies": {
+        "@types/react": "18.2.0",
+        "@types/react-dom": "18.2.0",
+        "@workleap/eslint-plugin": "1.2.0",
+        "@workleap/typescript-configs": "2.1.0",
+        "react": "18.2.0",
+        "react-dom": "18.2.0",
+        "react-router": "6.11.1",
```

Comment:
react-router is not in the peer dependencies list. should it be removed from dev dependencies, or added to the peer dependencies?